server/core/region_option.go

```diff
@@ -39,8 +39,9 @@ func WithDownPeers(downPeers []*pdpb.PeerStats) RegionCreateOption {
 
 // WithFlowRoundByDigit set the digit, which use to round to the nearest number
 func WithFlowRoundByDigit(digit int) RegionCreateOption {
+	flowRoundDivisor := uint64(math.Pow10(digit))
```

Comment:
no need?

Reply:
Reduce the number of `math.Pow10`.

Reply:
Now the return value (as func) of WithFlowRoundByDigit will be called a lot.